src/Libraries/RevitNodesUI/RevitDropDown.cs

```diff
@@ -478,7 +478,10 @@ protected override SelectionState PopulateItemsCore(string currentSelection)
                 if (category != null)
                 {
                     string name = getFullName(category);
-                    Items.Add(new DynamoDropDownItem(name, categoryId));
+                    if (name != "Cameras")
```

Comment:
I think just because you can't do anything with them today is not a particularly great reason to hide them, but I have no insight one way or the other. - wonder what @sm6srw thinks.